luigi_pipeline/lib/model/mito_mt_schema.py

```diff
@@ -0,0 +1,113 @@
+import hail as hl
+
+from lib.model.base_mt_schema import row_annotation
+from lib.model.seqr_mt_schema import BaseSeqrSchema, SeqrGenotypesSchema, SeqrVariantsAndGenotypesSchema
+
+
+class SeqrMitoVariantSchema(BaseSeqrSchema):
+
+    def __init__(self, *args, high_constraint_region=None, **kwargs):
+        super().__init__(*args, **kwargs)
+        self._high_constraint_region = high_constraint_region
+
+    # Mitochondrial only fields
+    @row_annotation()
+    def gnomad_mito(self):
+        return self._selected_ref_data.gnomad_mito
+
+    @row_annotation(disable_index=True)
+    def mitomap(self):
```

Comment:
So we do not currently allow you to filter on the mito-specific pathogenicity and predictor scores, but in the future we might and if we disable indexing for these fields we would need to reload all the data before we could do that. I think that, although we have no short-term plans to add filtering on those fields, the overhead of reloading data is so much worse than the overhead of a slightly larger index, especially given that the mito index is relatively small, that it makes sense for us to enable indexing for fields we think we may someday want. Therefore, we should enable indexing for all the mito-specific fields. The core seqr fields with indexing disabled should still be disabled here